src/routes/(root)/orders/success/+layout.svelte

```diff
@@ -12,5 +12,7 @@
 
 <slot />
 <div class="mt-24">
+	<!-- svelte-ignore a11y-missing-content -->
+	<a name="newsletter"></a>
```

Comment:
why do we have an empty link?

Reply:
I added this as an in-page anchor to which is referenced higher in the success page to scroll down to the sign-up. The empty "sign-up for the newsletter" bullet had no action to it.